### PR TITLE
fix: [LI-473] integrate new api for down ical

### PR DESCRIPTION
**Pull request by bennyli519**
### Issue ticket number and link
[LI-479](https://predictivehire.atlassian.net/browse/LI-479)
### Figma Screenshot
image

### Describe main changes
- implement GenerateICalInfo GraphQL
- remove axios

### Checklist in Local 
- [x] I have extract component business logic into custom hook
- [x] I have added unit tests
- [x] I have integrated with backend api and done a e2e test locally
- [x] I have updated the graphql query types

#### Screenshots for Integration Test in Local

https://github.com/user-attachments/assets/bcf91274-7669-4790-95b5-6339e6acf318

### Checklist in Dev
- [x] I have deployed to dev
- [x] I have integrated with backend api and done a e2e test

#### Screenshots for Integration Test in dev
image

### Checklist before requesting a review
- [ ] I have done the **code format** using `pnpm lint:fix` and `pnpm format:fix`
- [ ] I have **rebase squash** current branch
- [ ] I have confirmed **Naming Convention** with team if needed.
- [ ] I have done a **self review** in GitHub.

### Checklist in Qa Environment
- [ ] I have deployed to qa and let QA team do a test
- [ ] I have fix the QA bugs after QA team finish testing
- [ ] I have put the screenshot for Qa environment integration test below:

### Checklist in Sandbox Environment
- [ ] I have deployed to sandbox and let QA team do test
- [ ] I have fix the Sandbox Bugs after QA team finish testing
- [ ] I have put the screenshot for Qa environment integration test below:

### Checklist after Merge
- [ ] I have known the dependencies of release deployment of current PR.
- [ ] I have reviewed the release pipe from team lead to secure the production deployment.
- [ ] I have update the Jira tickets status after all task has been finished.

[LI-479]: https://predictivehire.atlassian.net/browse/LI-479?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/schedule-interview/hooks/useGenerateEventICalInfo.ts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+  const rollbar = useRollbar()
+  const { message } = App.useApp()
+
+  const [generateEventICalInfo, { loading: isGenerateEventICalInfoLoading }] = useMutation<
+    LiGenerateEventICalInfoMutation,
+    LiGenerateEventICalInfoMutationVariables
+  >(LIGenerateEventICalInfo, {
+    onError: (error) => {
+      void message.error("Failed to add calendar")
+      rollbar.error("Failed to add calendar", error)
```
Comment (yingzhi-lu) [inline src/pages/schedule-interview/hooks/useGenerateEventICalInfo.ts:19]: I am checking with Charmaine in Figma about the error message.

**Review by yingzhi-lu — APPROVED** [review]
LGTM!

File: src/__generated__/gql.ts [hunk 2]
```diff
@@ -15,6 +15,7 @@ import { TypedDocumentNode as DocumentNode } from '@graphql-typed-document-node/
 const documents = {
     "\n  mutation LIBookCandidateInterview($input: LIBookCandidateInterviewDto!) {\n    LIBookCandidateInterview(input: $input) {\n      jobRequisitionName\n      interviewType\n      interviewLocation\n      interviewLink\n      timezone\n      interviewStartsAt\n      interviewEndsAt\n      iCalId\n    }\n  }\n": types.LiBookCandidateInterviewDocument,
     "\n  mutation LICancelCandidateInterview($input: LICancelCandidateInterviewDto!) {\n    LICancelCandidateInterview(input: $input) {\n      interviewEventId\n      interviewScheduleId\n      interviewUniqueName\n      interviewDisplayName\n      interviewType\n      interviewMode\n      status\n    }\n  }\n": types.LiCancelCandidateInterviewDocument,
+    "\n  mutation LIGenerateEventICalInfo($input: LIGenerateEventICalDto!) {\n    LIGenerateEventICalInfo(input: $input) {\n      iCalContent\n    }\n  }\n": types.LiGenerateEventICalInfoDocument,
```
Comment (yingzhi-lu) [inline src/__generated__/gql.ts:18]: By the way I noticed scheduler is failing due to fileUrl has been deleted, could you double check the query is updated in scheduler-link repo? Thanks

Comment (bennyli519) [reply src/__generated__/gql.ts:18]: updated thanks

**Pull request merged**